lib/addons/p5.dom.js

```diff
@@ -784,6 +784,81 @@
     return self;
   };
 
+  /**
+   * Creates a colorPicker element in the DOM for color input.
+   * The .value() method will return a hex string (#rrggbb) of the color.
+   * The .color() method will return a p5.Color object with the current chosen color.
+   *
+   * @method createColorPicker
+   * @param {String|p5.Color} [value] default color of element
+   * @return {p5.Element} pointer to <a href="#/p5.Element">p5.Element</a> holding created node
+   * @example
+   * <div>
+   * <code>
+   * var inp1, inp2;
+   * function setup() {
+   *   createCanvas(100, 100);
+   *   background('grey');
+   *   var c1 = '#ff0000';
+   *   inp1 = createColorPicker(c1);
+   *   var c2 = color('yellow');
+   *   inp2 = createColorPicker(c2);
+   *   inp1.input(setShade1);
+   *   inp2.input(setShade2);
+   * }
+   *
+   * function draw() {
+   *   // Finding a shade between the two
+   *   var commonShade = lerpColor(inp1.color(), inp2.color(), 0.5);
```

Comment:
to create a new color ever loop of draw will actually slow things down quite a bit when there's more going on in the sketch. is it possible to make an example using a `.changed()` listener instead? this would fit better with best JS practices, setting the user up for good programming patterns. see [this example](https://p5js.org/reference/#/p5.Element/changed) as a guide.

Reply:
Will shift that code to the Input Change callback methods, setShade1 and setShade2 here!